smart-contracts/Cargo.toml

```diff
@@ -15,6 +15,10 @@ members = [
   "contracts/cl-vault",
   "contracts/merkle-incentives",
   "contracts/range-middleware",
+  "contracts/cw-vault-standard",
+  "contracts/cw-vault-multi-standard",
+  "packages/interface"
```

Comment:
I used the other packages as workspace-dependencies. Unfortunately, I don't recall the issue, but was having some in the past, so moved them out of this array. If there is not a strong argument for keeping it here, I'd prefer to also do that.